integration_testing/features/coach/coach-can-edits-group.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Edit a group’s name
+    Coach should be able to sign in to access Kolibri Coach tab
+    Coach can edit a group's name
+
+  Background:
+    Given you are in coach groups page
+    Given that there are groups created
+
+  Scenario: Edit a group name
+    When I click on OPTIONS for a group
```

Comment:
Another strategy we should agree on is how to distinguish the UI elements in writing. I would favor capitalizing just the first letter (even when they appear fully capitalized in the UI, for readability purposes), and enclosing the UI element string inside `**` chars. That is `*Option*` instead of `OPTIONS` as you proposed. 

On another hand I also like the `*Plugin name > Tab name*` convention for indicating the current position of the user in Kolibri, that is `*Coach > Groups*`, `*Learn > Recommended*`, `*Facility > Settings*`, `*Device > Info*` and so forth... 
@jonboiser  @indirectlylit thoughts?